packages/sdk/client/src/worker/onconnect.ts

```diff
@@ -57,6 +57,11 @@ export const onconnect = async (event: MessageEvent<any>) => {
   const systemChannel = new MessageChannel();
   const appChannel = new MessageChannel();
 
+  // set locally set log configuration forwarded from localStorage
+  // TODO(nf): block worker initialization until this is set? we usually win the race.
+  port.onmessage = (event) => {
+    (globalThis as any).localStorage_dxlog = event.data.dxlog;
```

Comment:
maybe worth wrapping this in some condition to check that this is actually a dxlog message?

Reply:
Considered that, but I wasn't sure what that would look like, it later gets parsed as JSON and now at least prints a log if that fails, but I think generally it'll only be used if you know what you're doing.

Had also considered mirroring all of localStorage, or how else this initial message could be used, but couldn't justify making it more complicated or what would stand in the way of changing it if it turns out to be too naive.